PayButton/paybutton-server
# [Issue #189] feat: add xec client

Description:
Turns `bchdService` into `grpcService` and makes it so that it accepts both XEC and BCH addresses. The environment variable `GRPC_NODE_URL` was replaced by `GRPC_BCH_NODE_URL` and `GRPC_XEC_NODE_URL`,
 

Test Plan:
1. Edit the `.env` file to include a BCH and XEC node, example:
```
GRPC_BCH_NODE_URL=bchd.fountainhead.cash:443  
GRPC_XEC_NODE_URL=grpc.fabien.cash:8335   
```
2. Run `make reset-dev`
3. After the containers are up, try:
```
curl http://localhost:3000/api/transactions/sync/qzqh7ej2vz26a9xaxq7capzfwgxt5gem9g8rvfxc5t
curl http://localhost:3000/api/transactions/sync/qrmm7edwuj4jf7tnvygjyztyy0a0qxvl7quss2vxek
``` 
4. Now, after logging in to the dev user, go to http://localhost:3000/buttons and check that the  transactions are visible there.

Depends on:

- [x] #201
